src/TransitionDuration.h

```diff
@@ -0,0 +1,22 @@
+/*
+ Copyright 2017-present The Material Motion Authors. All Rights Reserved.
```

Comment:
You can move this protocol into MDMTransition.h to keep the Transition protocol family together.